src/gitea/issue/async_issue.py

```
@@ -0,0 +1,277 @@
+
+from aiohttp import ClientResponse
+
+from gitea.issue.base import BaseIssue
+from gitea.resource.async_resource import AsyncResource
+from gitea.utils.response import process_async_response
+
+
+class AsyncIssue(BaseIssue, AsyncResource):
+    """Asynchronous Gitea Issue resource."""
+
+    async def _list_issues(  # noqa: PLR0913
```

Ruff (0.14.13)

Unused `noqa` directive (non-enabled: `PLR0913`)

Remove unused `noqa` directive

(RUF100)